### PR TITLE
Update query-container.md

### PR DESCRIPTION
**Pull request by helen-laktionova**
Add an info block with soft deprecation

## PR Description

TBD

## Checklist
- [ ] I agree with the Code Contribution License Agreement in CONTRIBUTING.md

### CODE REVIEW

**Review by dereuromark — APPROVED** [review]

**Review by gerner-spryker — APPROVED** [review]

File: docs/scos/dev/back-end-development/zed/persistence-layer/query-container/query-container.md [hunk 1]
```diff
@@ -24,6 +24,13 @@ redirect_from:
   - /v1/docs/en/query-container
 ---
 
+{% info_block infoBox "Soft deprecation" %}
+
+Query containers are soft deprecated.
```
Comment (PhilinTv) [inline docs/scos/dev/back-end-development/zed/persistence-layer/query-container/query-container.md:29]:
```suggestion
Query containers MUST NOT be used to cross module boundaries, as it increases modules coupling. However, they can be used behind Repository and Entity managers as query aggregations.
```

File: docs/scos/dev/back-end-development/zed/persistence-layer/query-container/query-container.md [hunk 2]
```diff
@@ -24,6 +24,13 @@ redirect_from:
   - /v1/docs/en/query-container
 ---
 
+{% info_block infoBox "Soft deprecation" %}
```
Comment (PhilinTv) [inline docs/scos/dev/back-end-development/zed/persistence-layer/query-container/query-container.md:27]:
```suggestion
{% info_block infoBox "When to use Query Containers" %}
```

File: docs/scos/dev/back-end-development/zed/persistence-layer/query-container/query-container.md [hunk 3]
```diff
@@ -24,6 +24,13 @@ redirect_from:
...
+
+Query containers are soft deprecated.
+It is recommended to use [Repository](/docs/scos/dev/back-end-development/zed/persistence-layer/repository.html) and [Entity Manager](/docs/scos/dev/back-end-development/zed/persistence-layer/entity-manager.html) instead.
```
Comment (PhilinTv) [inline docs/scos/dev/back-end-development/zed/persistence-layer/query-container/query-container.md:30]:
```suggestion
```

File: docs/scos/dev/back-end-development/zed/persistence-layer/query-container/query-container.md [hunk 4]
```diff
@@ -24,6 +24,13 @@ redirect_from:
   - /v1/docs/en/query-container
 ---
 
+{% info_block infoBox "Soft deprecation" %}
+
+Query containers are soft deprecated.
+It is recommended to use [Repository](/docs/scos/dev/back-end-development/zed/persistence-layer/repository.html) and [Entity Manager](/docs/scos/dev/back-end-development/zed/persistence-layer/entity-manager.html) instead. 
+
```
Comment (PhilinTv) [inline docs/scos/dev/back-end-development/zed/persistence-layer/query-container/query-container.md:31]:
```suggestion
Previously Query Containers were used to cross-module borders (via Dependency Providers), which led to higher module coupling and leaking of persistence layer from one domain object to another and therefore higher maintenance efforts and lower code reusability. This usage has been deprecated and is not recommended in project development.
```

**Review by PhilinTv — CHANGES_REQUESTED** [review]
--

**Pull request merged**